core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -121,7 +121,7 @@ public String description() {
   @Override
   public void execute(final String[] args) throws Exception {
     Opts opts = new Opts();
-    opts.parseArgs(PrintInfo.class.getName(), args);
+    opts.parseArgs("accumulo rfile-info", args);
```

Comment:
How did this work before?  I have used just "accumulo rifile-info <rfile>" without the classname...